airflow/providers/neo4j/operators/neo4j.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Neo4JOperator to interact and perform action on Neo4J graph database.
+This operator is designed to use Neo4J Hook and the
+Python driver: https://neo4j.com/docs/api/python-driver/current/
+"""
+
+from os.path import isfile
+
+from neo4j import BoltStatementResult
+
+from airflow.exceptions import AirflowException
+from airflow.models import BaseOperator
+from airflow.providers.neo4j.hook import Neo4JHook
+from airflow.utils.decorators import apply_defaults
+
+
+class Neo4JOperator(BaseOperator):
```

Comment:
I think the name is a bit confusing. The operator doesn't perform action on Neo4J (like running query, creating table etc..) but move data from Neo4J to CSV.  Usually such operators have naming convention of `x To y` Maybe `Neo4JOperatorDownloadOperator` ? (But that is just my thought maybe others will disagree)

Reply:
Sure, there were plans to make input & output operators and support different foramts. However I dont think its viable to develop a complete feature set in the first iteration.

I'll capture your thoughts in the JIRA ticket.